admin/app/routes/authenticated/organizations/get/campaigns.js

```diff
@@ -14,6 +14,11 @@ export default class OrganizationCampaignsRoute extends Route {
       'page[number]': params.pageNumber || 1,
       'page[size]': params.pageSize || 10,
     };
-    return this.store.query('campaign', query);
+    const campaigns = await this.store.query('campaign', query);
+
+    return {
+      campaigns,
+      organizationId,
+    };
```

Comment:
**suggestion**
Si j'ai bien compris on file l'organizationId surtout pour la page `all-tags` ici ?
Mais dans la PR qui arrive sur les tags (2601) on refacto le model de la route `all-tags` pour que la route ait le model `organization`. Du coup je propose qu'on laisse le `return this.store.query('campaign', query);` plutôt que renvoyer un model plus complexe ici et qui ne sera plus utile pour la suite.
Et du coup on changerait le nom de la route `all-tags` dans la PR suivante.

Reply:
Je vois pas le lien 'campaigns organizationId' avec la page des all-tags, preneur d'explication. Pour la page all-tags elle le récupère depuis la route all-tags ? du coup oui comme on le modifie dans l'autre PR on va le récupérer directement.

Reply:
Ce changement concerne le titre de la page `/campaings`